### PR TITLE
Fix exception when client does not start properly.

### PR DESCRIPTION
**Pull request by ehuss**

### CODE REVIEW

**Comment by rwols** [comment]
This is a pet-peeve of mine but we keep talking about *clients* while we should be talking about *servers*.

**Comment by tomv564** [comment]
@ehuss thanks for your contribution!

This project uses the `mypy` type checker, for which I need to write some contributor documentation.

The `mypy` type checker does not like applying `bool()` to client, but changing to the same check as used in other commands should work:
```
if client and client.has_capability('codeActionProvider'):
    return True
```

File: main.py [hunk 1]
```diff
@@ -1929,7 +1929,7 @@ class LspCodeActionsCommand(sublime_plugin.TextCommand):
     def is_enabled(self, event=None):
         if is_supported_view(self.view):
             client = client_for_view(self.view)
-            return client and client.has_capability('codeActionProvider')
+            return bool(client) and client.has_capability('codeActionProvider')
```
Comment (tomv564) [inline main.py]: Unfortunate typing issue with mypy, but can you change the syntax to:
```
if client and client.has_capability('codeActionProvider'):
    return True
```

**Comment by tomv564** [comment]
I merged the fix manually as I was unable to push the mypy fix to your branch. 
Thanks!

**Pull request closed without merging**